src/algorithms/crypto/randomness.md

```diff
@@ -86,6 +66,36 @@ GetRandomness(dst, l, s):
 {{< readfile file="/docs/actors/actors/crypto/randomness.go" code="true" lang="go" >}}
 {{< readfile file="/docs/systems/filecoin_blockchain/struct/chain/chain.go" code="true" lang="go" >}}
 
+## Drawing tickets from the VRF-chain for proof inclusion
+
+There are a few instances in which the protocol requires inclusion  to be drawn from the Filecoin blockchain's VRF-chain (which generates {{<sref tickets>}} with each new block), in order to tie certain proofs to a particular set of Filecoin blocks (i.e. a given chain or fork). This is notably the case for Seal Pre-commits (see {{<sref sealing>}}).
```

Comment:
the protocol requires inclusion?

Reply:
"This is notably the case for Seal Pre-commits (see {{<sref sealing>}})." is awkward phrasing

Reply:
this is the fact that SealRandomness must be taken from VRF chain, this ensures that no other fork can replay it.

Reply:
fair enough. I'll rephrase.